src/Illuminate/Database/Query/Grammars/MariaDbGrammar.php

```diff
@@ -53,4 +53,19 @@ public function useLegacyGroupLimit(Builder $query)
     {
         return false;
     }
+
+    /**
+     * Apply custom ordering to a query based on a priority array.
+     *
+     * @param  $column
+     * @param  array  $priority
+     * @param  $direction
+     * @return string
+     */
+    public function orderByPriority($column, array $priority, $direction = 'asc')
```

Comment:
This implementation is not necessary: `MariaDbGrammar` extends `MySqlGrammar` and so inherits the method.

Reply:
Correct. I have removed the code from MariaDbGrammar.